src/lib/dummy-rows.component.css

```diff
@@ -0,0 +1,10 @@
+host {
```

Comment:
`.super-table-dummy-rows`

`<div class="super-table-dummy-rows"></div>` in the template.